portfolio/src/main/webapp/index.html

```diff
@@ -10,9 +9,8 @@
   <link rel="stylesheet" href="style.css"> 
   <script src="https://www.gstatic.com/charts/loader.js"></script>
   <script src="script.js"></script>
-  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAryuMWR7Zsz4IOnEvMX7lksuZr6mpYkuw"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=API_KEY"></script>
```

Comment:
@lgsds - do you think they need to hide this API key in addition to the perspective key?  I think for Maps, there is a way they can restrict the key so that it's only usable on specific websites (https://developers.google.com/maps/documentation/embed/get-api-key).  Not for this PR but I wonder if something like that could be possible for Perspective

Reply:
It's fine ([and almost unavoidable really, without a server-side solution](https://stackoverflow.com/questions/1364858/what-steps-should-i-take-to-protect-my-google-maps-api-key)) to expose the key for Google Maps. My understanding is that API keys can be used for multiple Google APIs (e.g. both Maps and Perspective), so we'll definitely want to set some restrictions.

Domain restrictions (to localhost and the deployed app) are probably the easiest route. The only caveat is that enabling the key for localhost (for local development) means that technically anyone can also use the key for localhost. We could also try limiting to IP addresses, but that's a bit more involved.

Reply:
SGTM, we won't worry about this for now.